package.json

```diff
@@ -17,6 +17,7 @@
   },
   "homepage": "https://github.com/samples-afaq-tariq/node-backend-test#readme",
   "dependencies": {
-    "hapi": "^17.6.0"
+    "hapi": "^17.6.0",
+    "joi": "^13.7.0"
```

Comment:
I committed `package-lock.json` from early on under the impression that it would be the lock-file used by all npm install operations from that point forward. Is that not the case? Or are you referring to something else I'm missing here?